DonjinKrawler-Client/src/main/java/donjinkrawler/Player.java

```
@@ -59,16 +65,22 @@ public Player(PlayerData playerData, Client client) {
         this.musicMaker = new MusicMaker();
 
         data = playerData;
+        itemVisitor = new ItemVisitorImpl(data, inventory, armorBonus, weaponBonus);
         loadImage();
     }
 
     private void loadImage() {
-        ImageIcon ii = new ImageIcon(ClassLoader.getSystemResource("craft.png").getFile());
-        image = ii.getImage();
-        height = image.getHeight(null);
-        width = image.getWidth(null);
-        ii = new ImageIcon(ClassLoader.getSystemResource("attack.png").getFile());
-        attackIMG = ii.getImage();
+        try {
+            InputStream stream = getClass().getResourceAsStream("/craft.png");
+            ImageIcon ii = new ImageIcon(ImageIO.read(stream));
+            image = ii.getImage();
+            height = image.getHeight(null);
+            width = image.getWidth(null);
+            ii = new ImageIcon(ImageIO.read(getClass().getResourceAsStream("/attack.png")));
+            attackIMG = ii.getImage();
+        } catch (Exception ignored) {
```

If possible fix this.

From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Don't ignore exceptions.`